### ISSUE TITLE
Music Louge: doesn't work without sound

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#214]
There is a rule in the accessibility guidelines about being able to pause sound (mostly for screen reader use), but also in this case the demo doesn't really work without sound. Not sure if there is a way to indicate something is happening for deaf/hard of hearing users through a difference means than just audio. There are flashing lights at the moment...maybe they could be more obvious of a change when different cubes are added.
